stream-chat-android-ui-components/src/main/kotlin/io/getstream/chat/android/ui/message/input/MessageInputView.kt

```diff
@@ -178,14 +178,6 @@ public class MessageInputView : ConstraintLayout {
         refreshControlsState()
     }
 
-    @Deprecated(
```

Comment:
Added a note for myself to refactor the boilerplate around this block that was only necessary to support external SuggestionListView

Reply:
👍 could you add a task so we won't forget about that?